### PR TITLE
remove some duplication between pages around events and commands

### PR DESCRIPTION
**Pull request by claymccoy**
There was some discussion about moving this up one level and out of the developer group. But I didn't want to make changes and move it at the same time. It may be okay where it is, there are several links that would need to change.

### CODE REVIEW

File: docs/developer/quick-start.md [hunk 1]
```diff
@@ -87,71 +61,26 @@ that looks something like this:
...
 Event handlers react to GitHub pushes, updates to issues, pull
 requests, or similar events.
 
-To create an event handler, you provide:
-
--   the type of event you want to react to
--   the code that implements your event handler
-
-This event handler is triggered when a commit is pushed
+Let's add an event handler notifies us in Slack when a commit is pushed
```
Comment (ddgenome) [inline docs/developer/quick-start.md]: "event handler **that** notifies us"?

File: docs/developer/quick-start.md [hunk 2]
```diff
@@ -198,6 +113,3 @@ The `handle` code in this example
 [command]: commands.md (Atomist Command Automations)
 [event]: events.md (Atomist Event Automations)
 [graphql-api]: graphql.md (Atomist Automation GraphQL)
-
```
Comment (ddgenome) [inline docs/developer/quick-start.md]: Is the link for `[slack]` somewhere else in the file?

File: docs/developer/quick-start.md [hunk 3]
```diff
@@ -5,53 +5,27 @@ Develop your first automations: a bot command and an event handler.
...
 intent to the Atomist bot, then collects the required
 parameters, and invokes the code implementing your command.
 
-This bot command searches [Stack Overflow][so] and replies with the results:
-
-```typescript
-@CommandHandler("Query Stack Overflow", "search so")
-@Tags("stack-overflow")
-export class SearchStackOverflow implements HandleCommand {
+Let's try it out a bot command that searches [Stack Overflow][so] and replies with the results:
```
Comment (WillGardella) [inline docs/developer/quick-start.md]: Let's try out (no it)

File: docs/developer/quick-start.md [hunk 4]
```diff
@@ -87,71 +61,26 @@ that looks something like this:
...
-        } else {
-            return Promise.resolve(Success);
-        }
-    }
-}
-```
+a string like "Crushed #77!".
 
-As above, follow the standard procedure for getting a TypeScript/JavaScript going:
+This code is in a different automation client, but we get it and run it the same way:
```
Comment (WillGardella) [inline docs/developer/quick-start.md]: you instead of we

File: docs/developer/quick-start.md (hunk 2 above)
Comment (claymccoy) [reply docs/developer/quick-start.md]: yes

File: docs/developer/quick-start.md (hunk 1 above)
Comment (claymccoy) [reply docs/developer/quick-start.md]: "that" is correct

**Pull request merged**